public/app/features/admin/UserOrgs.tsx

```diff
@@ -133,8 +134,21 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps> {
   state = {
     currentRole: this.props.org.role,
     isChangingRole: false,
+    roleOptions: [],
+    builtInRoles: {},
   };
 
+  componentDidMount() {
+    if (contextSrv.licensedAccessControlEnabled()) {
+      if (contextSrv.hasPermission(AccessControlAction.ActionRolesList)) {
+        fetchRoleOptions(this.props.org.orgId).then((roles) => this.setState({ roleOptions: roles }));
```

Comment:
```suggestion
        fetchRoleOptions(this.props.org.orgId).then((roles) => this.setState({ roleOptions: roles })).catch(err => console.error(err));
```
Should probably handle an error here too.

Reply:
Yes!